packages/grid/backend/grid/start.sh

```diff
@@ -28,15 +21,17 @@ fi
 # only set by kubernetes to avoid conflict with docker tests
 if [[ ${DEBUGGER_ENABLED} == "True" ]];
 then
-    pip install --user debugpy
+    uv pip install debugpy
     DEBUG_CMD="python -m debugpy --listen 0.0.0.0:5678 -m"
 fi
 
-set +e
+source $APPDIR/.venv/bin/activate
```

Comment:
Q: Should the venv be activate before installation of debupy, as it would not be available for later commands?

Reply:
uv pip install works regardless if you've activated the environment or not